assets/js/utils/attributes.js

```diff
@@ -22,7 +22,7 @@ const attributeSettingToObject = ( attribute ) => {
 /**
  * Format all attribute settings into objects.
  */
-const attributeObjects = ATTRIBUTES.reduce( ( acc, current ) => {
+export const attributeObjects = ATTRIBUTES.reduce( ( acc, current ) => {
```

Comment:
I'm not sure to understand why is this exported, is it used somewhere else?

Reply:
ya let's leave this unexported for now (I did a search it's not used anywhere) push incoming.